site/src/scaffold/styles/layout.scss

```diff
@@ -108,6 +108,14 @@ $size-bar-width: 300px - strip-units($gap-size) * $document-font-size;
 
 .example {
 	padding: $layout-2;
+
+	& .example-lg {
+		height: 70px;
```

Comment:
What's up with this?

Reply:
This is no longer needed with the new examples, removed